examples/send_dates_transaction.js

```diff
@@ -0,0 +1,45 @@
+const pkg = require('../package.json');
+const { Exporter } = require('../index')
+const exporter = new Exporter(pkg.name, true)
+
+async function pushData() {
+  const timestamp = Math.floor(new Date() / 1000)
+
+  let lastPosition = await exporter.getLastPosition()
+  console.log(`Last position: ${JSON.stringify(lastPosition)}`)
+  let key = 1
+  if(lastPosition) {
+    key = lastPosition.key + 1
+  }
+
+  console.log("Sending data with transaction...")
+  await exporter.sendDataWithKey({
+    timestamp: timestamp,
+    iso_date: new Date().toISOString(),
```

Comment:
Fetching of the last position does take some time, so these 2 will end up pointing to different times. I suggest you have `const now = new Date()` at the top and use it both for the timestamp and iso_date

Reply:
Yes, storing exactly the same time better illustrate the purpose of the example, changed.